.github/workflows/test-images.yaml

```diff
@@ -1,14 +1,14 @@
 name: Create Test Images
 on:
   push:
-    branches:
-      - "staging"
     paths:
       - 'test/e2e/fixtures/Dockerfile.*'
```

Comment:
Is it worth adding the workflow to the paths - so if there is an update here it's automatically triggered?

Reply:
Here I tried running the action without path
which works fine as expected 
- [Action](https://github.com/sudharshanibm3/cloud-api-adaptor/actions/runs/5282462114)

Reply:
That wasn't my suggestion. I wanted you to add `.github/workflows/test-images.yaml` as another path, so if we update the workflow it would automatically run.

Reply:
You also won't need the `#test` comment in the Dockerfile if you add that rule.

Reply:
Added `.github/workflows/test-images.yaml` in paths